src/Components/ArtworkGrid/ArtworkGrid.tsx

```diff
@@ -15,14 +15,16 @@ import GridItem from "../Artwork/GridItem"
 
 type SectionedArtworks = Array<Array<ArtworkGrid_artworks["edges"][0]["node"]>>
 
+type Artwork = ArtworkGrid_artworks["edges"][0]["node"]
+
 export interface ArtworkGridProps
   extends React.HTMLProps<ArtworkGridContainer> {
   artworks: ArtworkGrid_artworks
   columnCount?: number | number[]
   preloadImageCount?: number
   itemMargin?: number
   mediator?: Mediator
-  onBrickClick?: () => void
+  onBrickClick?: (artwork?: Artwork) => void
```

Comment:
Added this optional param since in this callback, we do want to get some data about the artwork back (href and id added below), for tracking purposes.